var/spack/repos/builtin/packages/wannier90/package.py

```diff
@@ -5,6 +5,7 @@
 
 import inspect
 import os.path
+from llnl.util.filesystem import find
```

Comment:
You shouldn't need to import this, it should already be present

Reply:
D'oh! :)  I'll fix that soon!

Reply:
Got it!